SocialSharing/SocialSharingViewController.m

```diff
@@ -74,4 +75,54 @@ - (void)imagePickerControllerDidCancel:(UIImagePickerController *)picker {
     [self dismissViewControllerAnimated:YES completion:NULL];
```

Comment:
Use nil instead of NULL
Reason : http://stackoverflow.com/questions/1564410/when-should-i-use-nil-and-null-in-objective-c